lobster_simulator/sensors/Sensor.py

```diff
@@ -19,7 +19,8 @@
 
 class Sensor(ABC):
 
-    def __init__(self, robot: AUV, position: Vec3, orientation: Quaternion, time_step: SimulationTime):
+    def __init__(self, robot: AUV, position: Vec3, time_step: SimulationTime, orientation: Quaternion,
+                 noise_stds = None):
```

Comment:
Not sure if correct type
```suggestion
                 noise_stds: List[float] = None):
```

Reply:
It can also be a float, so I changed it to `Union[List[float], float]`